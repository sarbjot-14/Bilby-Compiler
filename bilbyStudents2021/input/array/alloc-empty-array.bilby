main{
	imm b := alloc [int](3+5);
	print b $n;
	
	imm c := alloc [float](3+5);
	print c $n;
	
	imm d := alloc [bool](3+5);
	print d $n;
	
	imm e := alloc [char](3+5);
	print e $n;
	
	
%	imm f := alloc [string](3+5);
%	print f $n;

	imm g := alloc [char](0);
	print g $n;

	

}