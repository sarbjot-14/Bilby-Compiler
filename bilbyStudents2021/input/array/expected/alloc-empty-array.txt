[0, 0, 0, 0, 0, 0, 0, 0]
[0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000]
[false, false, false, false, false, false, false, false]
[, , , , , , , ]
[]
